docs/options.md

```diff
@@ -216,3 +216,17 @@ Valid options:
 | Default      | CLI Override                                                | API Override                                                |
 | ------------ | ----------------------------------------------------------- | ----------------------------------------------------------- |
 | `"preserve"` | <code>--prose-wrap <always&#124;never&#124;preserve></code> | <code>proseWrap: "<always&#124;never&#124;preserve>"</code> |
+
+## EditorConfig
```

Comment:
Should be in `cli.md` I think.

Reply:
See https://github.com/prettier/prettier/pull/3255#discussion_r154543449

Reply:
Move docs to API's `resolveConfig` and some simple description to CLI's `--no-editorconfig`? `options.md` is for options from `.prettierrc`, which can be used in both CLI flag or API `prettier.format.options`.

Reply:
Oh, that makes much more sense. Fixed in https://github.com/prettier/prettier/pull/3255/commits/a36f9b9b577f3be370816b1fbbcd883d81da7657